lib/btc/BtcRpc.js

```diff
@@ -253,9 +253,9 @@ class BtcRpc {
     return this.asyncCall('listTransactions', [label, count, skip, inclWatchOnly]);
   }
 
-  async getRawTransaction({ txid }) {
+  async getRawTransaction({ txid, verbosity }) {
     try {
-      return await this.asyncCall('getRawTransaction', [txid, 1]);
+      return await this.asyncCall('getRawTransaction', [txid, verbosity || 1]);
```

Comment:
this will fallback to 1 for verbosity == 0. I suggest
```
async getRawTransaction({ txid, verbosity = 1 }) {
    try {
      return await this.asyncCall('getRawTransaction', [txid, verbosity]);
```

Reply:
Using `verbosity ?? 1` would be ideal, but breaks node 12 compatibility :disappointed: